taipy/gui_core/_adapters.py

```diff
@@ -228,7 +229,10 @@ def get(self):
                         datanode._editor_id,
                         _get_reason(is_readable(datanode), "Datanode not readable"),
                         _get_reason(is_editable(datanode), "Datanode not editable"),
-                        isinstance(datanode, _FileDataNodeMixin)
+                        isinstance(datanode, _FileDataNodeMixin),
+                        "Datanode doesn't have a file."
```

Comment:
```suggestion
                        "DataNode is not related to a file."
```

Reply:
But it is related to a file ... That is not there :-)